lib/bookstore-scrapper/indexTextbooks.mts

```diff
@@ -0,0 +1,51 @@
+/*
+    Run:
+    tsc --downlevelIteration --target es2021 --moduleResolution node --module es2022 --allowSyntheticDefaultImports indexTextbooks.mts; 
+    NODE_TLS_REJECT_UNAUTHORIZED='0' node indexTextbooks.mjs
```

Comment:
Can this be added as a command in our package.json?

Reply:
Have you taken a look at ts-node instead of tsc & node?

Reply:
Also, we should be able to avoid .mts/.mjs file extensions if we setup our environment correctly. I can take a look tomorrow.

Reply:
I tried different solutions including ts-node and updating the package.json and tsconfig files. Could not get any of them to work. This was the only solution that worked for me.

Reply:
Please take a look and let me know if there is anything else that I should be doing.

Reply:
Thanks for resolving this.